README.md

```diff
@@ -44,79 +62,75 @@ A [configs/](./configs/) directory contains primiary configuration text files fo
-- Water partitioned by the Schaake function to be soil moisture is placed into a conceptual linear reservoir that consists of two outlets that apply a minimum storage activation threshold.
-This activation threshold is identical for both outlets, and is based on an integral solution of the storage in the soil assuming Clapp-Hornberger parameters equal to those used in the NWM to determine that storage corresponding to a soil water content 0.5 m above the soil column bottom that produces a soil suction head equal to -1/3 atm, which is a commonly applied assumption used to estimate the field capacity water content.
-The first outlet calculates vertical percolation of water to deep groundwater using the saturated hydraulic conductivity of the soil multiplied by the NWM "slope" parameter, which when 1.0 indicates free drainage and when 0.0 indicates a no-flow lower boundary condition.
-The second outlet is used to calculate the flux to the soil lateral flow path, using a conceptual LKSATFAC-like calibration parameter.
-- The lateral flow is routed to the catchment outlet using a Nash-cascade of reservoirs to produce a mass-conserving delayed response, and eliminates the need for the 250 m lateral flow routing grid.
-- The groundwater contribution to base flow is modeled using either (a) an exponential nonlinear reservoir identical to the one in the NWM formulation, or (b) a nonlinear reservoir forumulation, which can also be made linear by assuming an exponent value equal to 1.0.
-
-**This code was written entirely by Fred L. Ogden, May 22-24, 2020, in the service of the NOAA-NWS Office of Water Prediction, in Tuscaloosa, Alabama.**
\ No newline at end of file
+2. Xinanjiang function (configuration: `surface_partitioning_scheme=Xinanjiang`). When using this runoff method the user must also include three parameters.
+
+If the **Xinanjiang** scheme is choosen, three parameters need to be included in the configuration file:
+1. a_Xinanjiang_inflection_point_parameter
+2. b_Xinanjiang_shape_parameter
+3. x_Xinanjiang_shape_parameter 
+
+## Rootzone-based Actual Evapotranspiration (AET)
+The user has the option to turn ON and OFF rootzone-based AET, default option is OFF. To turn it ON, the following parameters need to be included in the configuration file.
+1. aet_rootzone=true
+2. soil_layer_depths=0.1,0.4,1.0,2.0 (this is just an example)
+3. max_root_zone_layer=2
+
+## Soil freeze-thaw model (SFT)
+The Soil Freeze-Thaw (SFT) model is a standalone model.  For detailed information please refer to the [SFT repo](https://github.com/NOAA-OWP/SoilFreezeThaw). A few things to note when coupling SFT to CFE:
+1. SFT model provides `ice fraction` to CFE runoff schemes (Schaake `ice_fraction_schaake` and Xinanjiang `ice_fraction_xinan`)
+2. If the runoff scheme is Xinanjiang, no additional parameters are needed in the CFE config files. The framework will pass the parameter `ice_fraction_xinan` to CFE.
```

Comment:
It is unclear to me how to turn on SFT with the Xinanjiang scheme.

Reply:
SFT related instructions are provided on SFT repo, and there is a parameter called `ice_fraction_scheme` for it.